Snowflake.Data.Tests/SFSessionPropertyTest.cs

```diff
@@ -95,14 +120,16 @@ public static IEnumerable<TestCase> ConnectionStringTestCases()
                     { SFSessionProperty.USEPROXY, "false" },
                     { SFSessionProperty.PROXYHOST, "proxy.com" },
                     { SFSessionProperty.PROXYPORT, "1234" },
-                    { SFSessionProperty.NONPROXYHOSTS, "localhost" }
+                    { SFSessionProperty.NONPROXYHOSTS, "localhost" },
+                    { SFSessionProperty.BROWSER_RESPONSE_TIMEOUT, "120" }
```

Comment:
in test don't we want to have constant for this value?

Reply:
done